src/styles/styles.js

```diff
@@ -3437,6 +3437,10 @@ const styles = {
         marginTop: 15,
         textAlign: 'center',
     },
+
+    validateCodeTimer: {
```

Comment:
Actually, this style is a duplicate of `textBlue`, so let's use that instead of introducing a new one. There is this checklist item:

> If a new CSS style is added I verified that ... A similar style doesn't already exist

Reply:
Oops that's on me. Thanks @roryabraham

Reply:
Oops, I swear I looked for a matching style. Nice catch, @roryabraham!